packages/indexer-service/src/query-fees/allocations.ts

```diff
@@ -131,23 +131,21 @@ export class AllocationReceiptManager implements ReceiptManager {
                 id: receipt.id,
                 allocation: receipt.allocation,
                 signature: receipt.signature,
-                paymentAmount: receipt.paymentAmount,
+                fees: receipt.fees,
               },
               transaction,
             })
 
             // Don't save over receipts that are already advanced
             if (!isNew) {
-              const storedPaymentAmount = BigNumber.from(
-                state.getDataValue('paymentAmount'),
-              )
-              if (storedPaymentAmount.gte(receipt.paymentAmount)) {
+              const storedPaymentAmount = BigNumber.from(state.getDataValue('fees'))
```

Comment:
Rename to `storedFees`.
Might as well do a search for "payment" across the whole package while you're at it.

Reply:
Good catch, will do another pass.